1-mandatory/2-quotegenerator/quotes.js

```diff
@@ -490,3 +490,24 @@ const quotes = [
     author: "Zig Ziglar",
   },
 ];
+
+const quoteDiv = document.getElementById("quotes");
+let ourQuote = document.createElement("p");
+
+const authorDiv = document.getElementById("author");
+let ourAuthor = document.createElement("p");
+
+const randomQuote = pickFromArray(quotes);
+
+ourQuote.innerText = `${randomQuote.quote}`;
```

Comment:
the value of `randomQuote.quote` and `randomQuote.author` are already strings so you don't need to interpolate them

Reply:
Thanks Ellie you're right. I deleted them.